contrib/devtools/prepare_release/prepare_release_config.yaml

```diff
@@ -0,0 +1,79 @@
+# THIS IS A TEMPLATE FILE, FOR EXAMPLES ON ACTUAL RELEASES CHECK CONTENT OF prepare_release-X.Y.Z.yaml FILES
+# path to zend project root
+repository_root: "./"
+# new release version (format "major.minor.patch")
+version: ""
```

Comment:
Can we provide an example and also specify what to do when we want to release a version like `4.0.0-rc2`? Should I include `rc2` in the version field?

Reply:
Added details in the comment `suffix must not be included if beta or candidate`.